src/dso.scss

```diff
@@ -17,17 +17,19 @@ $library: true !default;
 @import "styles/mixins/bootstrap-variables";
 
 /*!
- * Bootstrap v3.3.7 (http://getbootstrap.com)
- * Copyright 2011-2016 Twitter, Inc.
- * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
- */
+* Bootstrap v3.3.7 (http://getbootstrap.com)
+* Copyright 2011-2016 Twitter, Inc.
+* Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+*/
 
 // Core variables and mixins
 @import "bootstrap-sass/assets/stylesheets/bootstrap/variables";
-@import "bootstrap-sass/assets/stylesheets/bootstrap/mixins";
+// @import "bootstrap-sass/assets/stylesheets/bootstrap/mixins";
+@import "styles/mixins/bootstrap-mixins";
 
 // Reset and dependencies
-@import "bootstrap-sass/assets/stylesheets/bootstrap/normalize";
+// @import "bootstrap-sass/assets/stylesheets/bootstrap/normalize";
+@import "styles/mixins/bootstrap-normalize-alt";
```

Comment:
Waarom `-alt` ? Schrijf het dan uit.

Maar het voegt niks toe, het is toch ook geen `bootstrap-mixins-alt` of `bootstrap-variables-alt` ?

Ik stem voor weghalen, anders dan een nieuwe schrijfwijze voegt het niets toe

Reply:
`dso-normalize`, dan maar.